bundles/org.eclipse.equinox.p2.metadata/src/org/eclipse/equinox/internal/p2/metadata/VersionParser.java

```diff
@@ -83,7 +83,7 @@ public static Version parse(String version, int start, int maxPos) throws Illega
 		}
 
 		if (!isLetter(c)) {
-			throw new IllegalArgumentException();
+			throw new IllegalArgumentException("Expected letter at position " + pos + " in version: " + version); //$NON-NLS-1$
```

Comment:
I know it might is a bit odd but as we have other exception messages localized, we should do this here as well (see for example line 131)

Reply:
Done